21.md

```diff
@@ -12,9 +12,27 @@ The scheme is `nostr:`.
 
 The identifiers that come after are expected to be the same as those defined in [NIP-19](19.md) (except `nsec`).
 
-## Examples
+#### Examples
 
 - `nostr:npub1sn0wdenkukak0d9dfczzeacvhkrgz92ak56egt7vdgzn8pv2wfqqhrjdv9`
 - `nostr:nprofile1qqsrhuxx8l9ex335q7he0f09aej04zpazpl0ne2cgukyawd24mayt8gpp4mhxue69uhhytnc9e3k7mgpz4mhxue69uhkg6nzv9ejuumpv34kytnrdaksjlyr9p`
 - `nostr:note1fntxtkcy9pjwucqwa9mddn7v03wwwsu9j330jj350nvhpky2tuaspk6nqc`
 - `nostr:nevent1qqstna2yrezu5wghjvswqqculvvwxsrcvu7uc0f78gan4xqhvz49d9spr3mhxue69uhkummnw3ez6un9d3shjtn4de6x2argwghx6egpr4mhxue69uhkummnw3ez6ur4vgh8wetvd3hhyer9wghxuet5nxnepm`
+
+### Linking HTML pages to Nostr entities
+
+`<link>` tags with `rel="alternate"` can be used to associate webpages to Nostr events, in cases where the same content is served via the two mediums (for example, a web server that exposes Markdown articles both as HTML pages and as `kind:30023' events served under itself as a relay or through some other relay). For example:
+
+```
+<head>
+  <link rel="alternate" href="nostr:naddr1qqyrzwrxvc6ngvfkqyghwumn8ghj7enfv96x5ctx9e3k7mgzyqalp33lewf5vdq847t6te0wvnags0gs0mu72kz8938tn24wlfze6qcyqqq823cph95ag" />
```

Comment:
The elements should have a `type` attribute, set to `application/nostr+json`, so clients don't have to infer the type by doing string matching on `href` values. This is standard practice for other types of alternate links in use on the Web today.

Reply:
I understand that this would be good to have in an ideal world but in the current world I don't think the `type` adds any value here. It cannot be relied upon by parsers anyway, so they will **necessarily** have to match on the `href` value. And it just complicates the spec and makes me (and presumably many others) less willing to read and implement it and memorize it (remember how I was implementing it on fiatjaf.com and got the string wrong?).

Also the most popular `<link>` tag in existence, the `rel="stylesheet"` one, doesn't use `type`. I guess people had the `text/css` in manuals before but browsers never cared for them as far as I know. Nowadays even the manuals have removed the `type`.

Reply:
That's because `rel=stylesheet` can only be a CSS stylesheet, so it's already specified by the rel itself.